INSTALL.rst

```diff
@@ -15,16 +15,6 @@ this:
 
    $ pip install invenio-rest[cors]
 
-Configuration
--------------
-
-===================== =================================================
-`REST_ENABLE_CORS`    Set to ``True`` to enable Cross-Origin Resource
-                      Sharing. Defaults to ``False``.
-`REST_CORS_RESOURCES` Dictionary for configuring CORS for endpoints.
-                      See Flask-CORS for further details.
```

Comment:
I agree with @drjova that it's better to keep them, so perhaps simply set their value in config.py